docs/browser_support.md

```diff
@@ -81,3 +81,12 @@ Examples for various installation methods:
 
       // Start loading FingerprintJS here
     ```
+## Syntax for Old browsers
```

Comment:
This is a requirement for all browsers. So please make the document have the following structure:

- Old browsers requirements
    - Polyfills
    - Code syntax